packages/db/package.json

```diff
@@ -38,29 +38,14 @@
   },
   "typesVersions": {
     "*": {
-      ".": [
-        "./index.d.ts"
-      ],
-      "types": [
-        "./dist/types.d.ts"
-      ],
-      "utils": [
-        "./dist/utils.d.ts"
-      ],
-      "runtime": [
-        "./dist/runtime/index.d.ts"
-      ]
+      ".": ["./index.d.ts"],
+      "types": ["./dist/types.d.ts"],
+      "utils": ["./dist/utils.d.ts"],
+      "runtime": ["./dist/runtime/index.d.ts"]
     }
   },
-  "files": [
-    "index.d.ts",
-    "virtual.d.ts",
-    "dist"
-  ],
-  "keywords": [
-    "withastro",
-    "astro-integration"
-  ],
+  "files": ["index.d.ts", "virtual.d.ts", "dist"],
+  "keywords": ["withastro", "astro-integration"],
```

Comment:
I feel like making this single-line is less readable. Maybe we should disable formatting for all `package.json` by default? I think package managers already format it on install.

Not a dealbreaker though if others are fine with this change. In that case, we can go ahead with it.

The other formatting looks great 👍